Lib/fontTools/ttLib/ttGlyphSet.py

```diff
@@ -178,12 +182,37 @@ def _setCoordinates(glyph, coord, glyfTable):
 
 
 class _TTVarGlyph(_TTGlyph):
-	def __init__(self, ttFont, glyphName, location):
+	def __init__(self, ttFont, glyphs, glyphName, location):
+
+		super().__init__(glyphs, glyphs[glyphName])
+
 		self._ttFont = ttFont
+		self._glyphs = glyphs
 		self._glyphName = glyphName
 		self._location = location
-		# draw() fills these in
-		self.width = self.height = self.lsb = self.tsb = None
+
+
+class _TTVarGlyphCFF(_TTVarGlyph):
+
+	def draw(self, pen):
+		varStore = self._glyphs.varStore
+		if varStore is None:
+			blender = None
+		else:
+			from fontTools.varLib.varStore import VarStoreInstancer
+			vsInstancer = VarStoreInstancer(varStore.otVarStore, self._ttFont['fvar'].axes, self._location)
```

Comment:
Would it be more efficient to store the instancer in the glyph set object, since the instancer has an internal cache?